### PR TITLE
Cigarette Cases + Wallet Tweak and Gold Engraved Lighter Resprite

### PR DESCRIPTION
**Pull request by TheKillfish**
## What Does This PR Do
Adds Cigarette Cases as avaliable items with three variants; normal, golden and suspicious. They are functionally identical to wallets except they cannot display IDs, cannot carry dice or screwdrivers, and are fireproof and acidproof. The normal case can be built at Protolathes (requiring Titanium and Materials 3), bought from hacked Cigarette Vendor for 100 credits or found as maints loot. The golden case is provided to NTRs as well as being a potential Cultural Artifact reward from the Bluespace Harvester. The suspicious case can be found as maints loot or bought from uplinks at 5TC in the Pointless Badassery section, as well as being provided in a new traitor Bundle.

The new Traitor bundle is called the TF13 Spy Bundle, and includes the following items:
- 357. Revolver
- 2 x Speedloaders
- Energy Dagger
- Syndicate Two Piece Suit
- Syndicate Cigarettes
- Suspicious Cigarette Case
- Stealth Box Implant
- Chameleon Kit
- Standard Zippo
- Syndicate Encryption Key

Other changes:
- The NTR's lighter has been resprited to look less shit
- Wallets have been tweaked to be able to carry Rolling Paper, but have had their capacities reduced to 8 for the Leather Wallet, and 4 for the Plastic Wallet.

## Why It's Good For The Game
New, flavourful items are fun and these give a wallet alternative, while the bundle is a TF2 referance and a spot of fun.
The wallet tweak was made to line up with the cases (and that logically you could fit them in a wallet, so why not?), and the lighter resprite was done because the current version looks butt ugly and I was already looking through sprites for inspiration.

## Images of changes
STILL WIP, WILL ADD LATER

## Testing
Put things in, hacked vendors, chilled as NTR, and became the Spy.

## Changelog
:cl:
add: Added three cigarette cases and a new Syndicate bundle
tweak: Made wallet able to store patches and rolling paper
tweak: Resprited the NTR's lighter to not look terrible anymore
imageadd: Added cigarette case sprites
/:cl:

### CODE REVIEW

**Comment by 1080pCat** [comment]
> The standard can be built at a Protolathe in the Equipment tab (requiring Materials Level 3 and 500 Titanium)

i'd much rather see this in the usual late. or only in hacked venders
could you also make the base one maints loot?

**Comment by 1080pCat** [comment]
on the bundle, i think replacing the clothing with a cham kit + agent id would make more sense

**Comment by AyIong** [comment]
image
Looks... flat?

**Comment by TheKillfish** [comment]
As for maints loot and normal autolathe: I see no reason why I can't, although I wanted it to be *slightly* harder to get since it gives Titanium.

As for the bundle: we already have the original Spy bundle that has that stuff, and no bundle uses the chameleon projector. A little bit of creative freedom can be had.

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -209,6 +209,24 @@
...
+
+/obj/item/storage/box/syndie_kit/bundle/tf13_spy
+	name = "TF13 Spy"
+	desc = "Brush up on your French and hone your backstabbing skills like a true, suave Gentlemann."
+	items = list(
+		/obj/item/gun/projectile/revolver, // 65TC
+		/obj/item/ammo_box/a357, // 15TC
+		/obj/item/ammo_box/a357, // 15TC
+		/obj/item/pen/edagger, // 10TC
+		/obj/item/chameleon, // 25TC
```
Comment (1080pCat) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:222]: with the suggestion of the cham kit, i'd also think the stealth box would make more sense

Comment (TheKillfish) [reply code/game/objects/items/weapons/storage/uplink_kits.dm:222]: Bugger me, the box was what I had in mind when brainstorming last night and I forgot about it

**Comment by 1080pCat** [comment]
> As for the bundle: we already have the original Spy bundle that has that stuff, and no bundle uses the chameleon projector. A little bit of creative freedom can be had.

bah saw this after the review.
that does make sense on the projector. i guess you could argue gun spy? but not having the ability to disguise in the tf2 spy reference kit doesn't make too much sense

**Comment by Fordoxia** [comment]
>Adds a new item, the cigarette case, and a subvariant called the golden cigarette case, along with a new Traitor bundle. It can carry broadly the same as wallets except for dice and screwdrivers, is fireproof (Golden is additionally acidproof), and has 15 storage slots (Golden has 20 slots), but cannot display IDs nor fit in the ID slot.

Bruh, 15 and 20 slots is way to much. 120 fucking trauma kits fit in there.

**Comment by Fordoxia** [comment]
> Looks... flat?

Cigarette cases IRL are also flat.

**Comment by McRamon** [comment]
so syndicate bundle comes with a cigarette case with NT logo on it?
also sprites need improvement, you can contact me on discord for help

**Comment by matttheficus** [comment]
The storage slot amount is a concern - wallets can already hold SILLY amounts of medical items. Completely against patches fitting inside them as well.

**Comment by TheKillfish** [comment]
Since I'm poking around in the code then, anything against me reducing storage space of wallets and the cig cases as a result?

File: code/game/objects/items/weapons/storage/wallets.dm [hunk 2]
```diff
@@ -118,4 +119,4 @@
 /obj/item/storage/wallet/cheap
 	name = "cheap wallet"
 	desc = "A cheap and flimsy wallet from the arcade."
-	storage_slots = 5		//smaller storage than normal wallets
+	storage_slots = 4		//smaller storage than normal wallets
```
Comment (DGamerL) [inline code/game/objects/items/weapons/storage/wallets.dm:122]: Same here

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 3]
```diff
@@ -209,6 +209,24 @@
 		/obj/item/clothing/head/collectable/slime,  // 0TC priceless
 		/obj/item/encryptionkey/syndicate) // 10TC
 
+/// 195 TC
+
```
Comment (DGamerL) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:213]:
```suggestion
```

File: code/game/objects/items/weapons/storage/cigcase.dm [hunk 4]
```diff
@@ -0,0 +1,45 @@
+/obj/item/storage/cigcase
+	name = "cigarette case"
+	desc = "A durable case for holding cigarettes and a variety of other things that can fit in it. You have a strange urge to crouch and look up when holding it."
+	storage_slots = 10
```
Comment (DGamerL) [inline code/game/objects/items/weapons/storage/cigcase.dm:4]: Your description says 15

File: code/game/objects/items/weapons/storage/cigcase.dm [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+	icon_state = "cigcase_nt"
+	origin_tech = "materials=5"
+	materials = list(MAT_GOLD = 500)
+
+/obj/item/storage/cigcase/syndie
+	name = "suspicious cigarette case"
+	desc = "Simple. Stylish. Shady. Perfect for the discerning ner-do-well who lurks in maintenance tunnels."
+	icon_state = "cigcase_syndie"
+	origin_tech = "materials=3;syndicate=1"
+	materials = list(MAT_TITANIUM = 250, MAT_PLASMA = 250)
```
[the remark this answers is not in the document]
Comment (TheKillfish) [reply code/game/objects/items/weapons/storage/cigcase.dm:45]: I still need to edit the main message to account for newer changes made

[the remark this answers is not in the document]
Comment (DGamerL) [reply code/game/objects/items/weapons/storage/cigcase.dm:45]: Sounds fair, please do that soon

**Comment by TheKillfish** [comment]
Going to be setting this to draft for now

**Comment by S34NW** [comment]
This PR has been marked as stale due to lack of activity, please continue working on the PR and/or address any comments within 7 days or we will have to close it.

**Comment by S34NW** [comment]
Bloody bot

**Comment by TheKillfish** [comment]
I'm just going to close this. Haven't been working on it in ages and I can't find motivation to continue working on it atm. Maybe I'll return to this, but that'll happen if and when it happens.

**Pull request closed without merging**